### PR TITLE
Decap config updates

### PR DESCRIPTION
**Pull request by grugnog**
- Drop path field
- Add site link and logo
- Add preview styles

----
:books: Documentation preview :books:: https://civicactions-handbook--1141.org.readthedocs.build/en/1141/

### CODE REVIEW

**Review by dmundra — APPROVED** [review]
Looks good to me. I was to able to preview it. I like the CSS changes!

File: admin/article.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  text-size-adjust: none;
+  line-height: 1.6;
+  font-size: 16px;
+  color: #000000de;
+}
+
+/* Hide the page title, but not the top level heading */
+div.frame-content > div:first-child > div:first-child > div:first-child > h1 {
+  display: none;
+}
```
Comment (dmundra) [inline admin/article.css:35]: Nice solution!

File: admin/article.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+th {
+  border-collapse: initial;
+  border-spacing: 0;
+  text-align: -webkit-left;
+
+  font-weight: 400;
+  vertical-align: top;
+}
```
Comment (dmundra) [inline admin/article.css]: We missing out on some table borders. If you want to add them for the row like material does that would be good.

Comment (grugnog) [reply admin/article.css]: OK, I think this should do it

**Comment by dmundra** [comment]
@grugnog looks good to me. Ready to merge.

**Pull request merged**